src/constants/constants.ts

```diff
@@ -0,0 +1,40 @@
+import { StatusCodes } from 'http-status-codes';
+
+export const VIEWS =  { BUILDER: {name: 'builder', hostname: 'builder.graasp.org'}, 
+                        PLAYER: {name: 'player', hostname: 'player.graasp.org'}, 
+                        EXPLORER: {name: 'explorer', hostname: 'explorer.graasp.org'},
+                        UNKNOWN: {name: 'unknown'}
+                      };
+
+export const ACTION_TYPES = { GET: 'get',
+                              GET_CHILDREN: 'get-children',
+                              UPDATE: 'update',
+                              CREATE: 'create',
+                              DELETE: 'delete',
+                              DOWNLOAD: 'download',
+                              COPY: 'copy',
+                              MOVE: 'move',
+                            };
+
+export const METHODS =  { GET: 'GET',
+                          POST: 'POST',
+                          PATCH: 'PATCH',
+                        }
+
+export const correctStatusCodes = [StatusCodes.OK, StatusCodes.CREATED, StatusCodes.ACCEPTED, StatusCodes.NON_AUTHORITATIVE_INFORMATION, StatusCodes.NO_CONTENT, StatusCodes.RESET_CONTENT, StatusCodes.PARTIAL_CONTENT, StatusCodes.MULTI_STATUS];
```

Comment:
I have not found something similar to the r`esponse.ok` in the reply. The Fastify reply object has the `reply.statusCode`, and then inside the Node `http.ServerResponse` we have the `statusMessage` attribute (but it is specific for each 201, 202...), so it would be the same.

Regarding the use of `http-status-codes`, it only provides the constants for each status code, so if we want to check all the 2xx cases, I think we have to set an array with all of them or something similar.